lib/runtime/src/config.rs

```diff
@@ -142,6 +177,22 @@ impl RuntimeConfigBuilder {
     }
 }
 
+/// Check if a string is truthy
+/// This will be used to evaluate environment variables or any other subjective
+/// configuration parameters that can be set by the user that should be evaluated
+/// as a boolean value.
+pub fn is_truthy(val: &str) -> bool {
+    matches!(val.to_lowercase().as_str(), "1" | "true" | "on" | "yes")
+}
+
+/// Check if a string is falsey
+/// This will be used to evaluate environment variables or any other subjective
+/// configuration parameters that can be set by the user that should be evaluated
+/// as a boolean value (opposite of is_truthy).
+pub fn is_falsey(val: &str) -> bool {
+    matches!(val.to_lowercase().as_str(), "0" | "false" | "off" | "no")
+}
+
```

Comment:
open issue to update parsing envs to ensure the env is either truthy or falsey, and error if is not in either list.

Reply:
@nnshah1 - to make an enchancement!